lib/streamlit/config.py

```diff
@@ -1052,89 +1052,116 @@ def _browser_server_port() -> int:
     categories=["theme"],
     description="""
         The preset Streamlit theme that your custom theme inherits from.
-        One of "light" or "dark".
+        This can be one of the following: "light" or "dark".
     """,
 )
 
 _create_theme_options(
     "primaryColor",
     categories=["theme", CustomThemeCategories.SIDEBAR],
-    description="Primary accent color for interactive elements.",
+    description="""
+        Primary accent color for interactive elements.
```

Comment:
Do we need the `interactive elements` here? The primary color is also used for many other parts besides interactive elements..